.changeset/tender-monkeys-beg.md

```diff
@@ -0,0 +1,5 @@
+---
+'@nhost/nhost-js': patch
+---
+
+chore(nhost-js): expose `url` in Functions and GraphQL client
```

Comment:
@plmercereau I know `url` was already exposed in the GraphQL client through `client.graphql.getUrl`, but it's `client.(auth|storage).url` in other packages. Not sure which way to pick though. Maybe it's better if we use a getter function everywhere.